python/ray/train/backend.py

```diff
@@ -67,8 +67,11 @@ def handle_failure(
 
         By default, restart all workers.
         """
+        logger.info("Shutting down all training workers.")
         worker_group.shutdown()
+        logger.info("Restarting all training workers.")
         worker_group.start()
+        logger.info("Setting up distributed backend on all workers.")
```

Comment:
Are these logs too verbose?

Reply:
Hmmm, most people don't run with log level DEBUG, and for very long running training jobs (15+ hours) I think these messages could be useful and it's not feasible to run the training job again.

cc @matthewdeng would also like to hear your thoughts here.

Reply:
I think INFO is fine given that this is only logged during failures, which should hopefully be rare.